editor/filesystem_dock.h

```diff
@@ -369,6 +369,9 @@ class FileSystemDock : public VBoxContainer {
 	static void _bind_methods();
 
 public:
+	HashMap<String, Color> get_folder_colors() const;
```

Comment:
```suggestion
	const HashMap<String, Color> &get_folder_colors() const;
```
I'd say this should be passed by reference, unless you absolutely need them to be copied